src/organizations/organizations.controller.ts

```diff
@@ -1,13 +1,26 @@
-import { Controller, Get, Post, Body, Patch, Param, Delete } from '@nestjs/common';
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Patch,
+  Param,
+  Delete,
+  UsePipes,
+} from '@nestjs/common';
 import { OrganizationsService } from './organizations.service';
 import { CreateOrganizationDto } from './dto/create-organization.dto';
 import { UpdateOrganizationDto } from './dto/update-organization.dto';
+import { TransformOrgDto } from './pipes/transform-org.pipe';
+import { ApiTags } from '@nestjs/swagger';
 
 @Controller('organizations')
+@ApiTags('organization')
```

Comment:
Do you want to pluralize (preferred) or leave as singular, but you must be consistent

Reply:
Noted.